scala3doc/dotty-docs/docs/blog/_posts/2017-09-07-third-dotty-milestone-release.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: blog-page
```

Comment:
Can we deleter the blog post file original blogpost file? This way git will consider it a move and we won't lose the history. Same for all other blog posts.

Reply:
We can't, these files are actually slightly different. We will do that when replacing Dottydoc as the tool for rendering the Dotty blog, which should be done soon-ish.